src/AppInstallerCommonCore/Experiment.cpp

```diff
@@ -13,35 +13,30 @@ namespace AppInstaller::Settings
     {
         ExperimentState GetExperimentStateInternal(Experiment::Key key, const UserSettings& userSettings)
         {
-            if (key == Experiment::Key::None)
-            {
-                return { true, ExperimentToggleSource::Default };
-            }
-
             if (!GroupPolicies().IsEnabled(TogglePolicy::Policy::Experiments))
             {
-                AICLI_LOG(Core, Info, <<
-                    "Experiment " << Experiment::GetExperiment(key).Name() <<
+                AICLI_LOG(Core, Info, << "Experiment " << Experiment::GetExperiment(key).Name() <<
                     " is disabled due to group policy: " << TogglePolicy::GetPolicy(TogglePolicy::Policy::Experiments).RegValueName());
                 return { false, ExperimentToggleSource::Policy };
             }
 
-            auto experiments = userSettings.Get<Setting::Experiments>();
+            if (key == Experiment::Key::None)

```

Comment:
Move this back to first.  While I'm not sure that we need a `None`, if we have one then checks against it should not end up logging about group policy being disabled.